lms/templates/help_modal.html

```diff
@@ -18,7 +18,7 @@
   <a href="#help-modal" rel="leanModal" role="button">${_("Help")}</a>
 </div>
 
-<section id="help-modal" class="modal" aria-hidden="true" role="dialog" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
+<section id="help-modal" class="modal" aria-modal="true" aria-hidden="true" role="dialog" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
```

Comment:
hmm, not sure who wrote the AC for this, but there is no such thing as an ```aria-modal``` attribute. It looks like this was recommended by the outside auditors, but this is a mistake.